.changelog/3605.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:fix
```

Comment:
Should this be `bug`? I can't remember if go-changelog accepts arbitrary release note categories or if it's fixed to a certain set.

Reply:
You're correct, fixed!